Formula/ooniprobe.rb

```diff
@@ -166,16 +166,57 @@ def install
     (var/"lib/ooni").mkpath
   end
 
-  def post_install
-    system bin/"ooniresources"
-    system bin/"oonideckgen", "-o", "#{HOMEBREW_PREFIX}/share/ooni/decks/"
```

Comment:
Oh, I see, you've just moved it further below. Apologies, that lost me momentarily. We'd prefer for it to be up here. `def plist` is supposed to be the second to last thing in a formula, before `test do` but after everything else.